internal/instruments.go

```diff
@@ -0,0 +1,38 @@
+package internal
+
+import (
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/metric"
+)
+
+type openTelemetryInstrumentation struct {
+	// Count of write commands performed
+	WriteCount metric.Int64Counter
+	// Count of new connections
+	NewConnectionsCount metric.Int64Counter
+}
+
+var Instruments = initInstruments()
+
+func initInstruments() *openTelemetryInstrumentation {
+	meter := global.Meter("github.com/go-redis/redis")
+
+	writeCount, err := meter.NewInt64Counter("redis.writes",
+		metric.WithDescription("the number of writes initiated"),
+	)
+	if err != nil {
+		Logger.Printf("failed to create instrument WriteCount")
+	}
+
+	newConnectionsCount, err := meter.NewInt64Counter("redis.connections",
```

Comment:
Perhaps `redis.dials`? `redis.connections` would be a good name for number of active connections. Or `redis.new_connections` and `redis.active_connections`

Reply:
Updated it to `new_connections`, thanks!